package-lock.json

```diff
@@ -32,7 +35,6 @@
         "micromatch": "^4.0.5",
         "normalize-path": "^3.0.0",
         "picocolors": "^1.0.0",
-        "postcss": "^8.4.20",
```

Comment:
[suggest] `postcss` moved `dependencies` to `devDependencies`. Could you restore it?